aten/src/ATen/native/mkldnn/Matmul.cpp

```diff
@@ -402,6 +409,70 @@ bool use_mkldnn_matmul(
   return (use_mkldnn_bf16_matmul(mat1, mat2, result) || use_mkldnn_fp16_matmul(mat1, mat2, result) || use_mkldnn_bf32_matmul(mat1, mat2, result));
 }
 
+void mkldnn_matmul_i8i8i32(
+    const Tensor &mat1,
+    const Tensor &mat2,
+    const Tensor &result) {
+  // x:s8 * w:s8 -> y:s32
+  TORCH_CHECK(
```

Comment:
Do we have to check a second time here ?
I suppose ` mkldnn_matmul_i8i8i32` is not an external API, right ?

Reply:
Thanks. Checks here are removed.